### PR TITLE
[TBCM-208] UI Enhancements + Reduced, but evenly spaced out elements

### PR DESCRIPTION
**Pull request by kushal-arora-fw**

### CODE REVIEW

File: apps/web/src/components/OccupationSelector.tsx [hunk 1]
```diff
@@ -50,8 +50,19 @@ export const OccupationSelector = ({ searchValue = '', showDescriptionModal = fa
...
           }
         />
-        <label className='font-bold' htmlFor={'selectAll'}>
+        <label className='' htmlFor={'selectAll'}>
           Select all
+          <span className='pl-1 font-bold'>
+            (
+            {
+              (
+                filteredOccupations?.filter((o: any) =>
```
Comment (dbayly-freshworks) [inline apps/web/src/components/OccupationSelector.tsx:59]: Nit - Do you have a type for this?

Comment (kushal-arora-fw) [reply apps/web/src/components/OccupationSelector.tsx]: Good catch! I wonder why I added any here. Type is inplicitly inferred "OccupationItemProps". Updated the code to reflect that.

**Review by dbayly-freshworks — APPROVED** [review]
Looks good to me

File: apps/web/src/components/RightSideBarActivities.tsx [hunk 2]
```diff
@@ -76,8 +76,16 @@ export const RightSideBarActivites: React.FC = () => {
           filteredData.every((a: any) => values.careActivities.includes(a.id))
         }
       />
-      <label className='font-bold' htmlFor={'selectAll'}>
+      <label htmlFor={'selectAll'}>
         Select all
+        <span className='pl-1 font-bold'>
+          (
+          {
+            (filteredData?.filter((d: any) => (values?.careActivities || []).includes(d.id)) || [])
```
[the remark this answers is not in the document]
Comment (kushal-arora-fw) [reply apps/web/src/components/RightSideBarActivities.tsx:97]: I have added an explanatory comment. However, it is missing type right from the API call - (old code). We anyway plan to make changes to the page, and any implementations - I anyway will add types everywheere.

**Pull request merged**